.circleci/config.yml

```diff
@@ -39,7 +39,22 @@ jobs:
           name: Test opening nteract desktop
           command: |
             yarn test
-            
+
+workflows:
+  version: 2
+  commit:
+    jobs:
+      - build
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
```

Comment:
It's usually friendly to other systems if you do repeated runs that are not "on the hour". 😄

Reply:
Wow I hadn't even considered that, good thinking! Will change 👍